config/locales/en.yml

```diff
@@ -2906,6 +2927,30 @@ en:
       showing_page: "Page %{page}"
       next: "Next »"
       previous: "« Previous"
+  user_mutes:
+    index:
+      title: "Muted Users"
+      my_muted_users: "My muted users"
+      you_have_muted_no_users: "You have not muted any users 🎉"
+      you_have_muted_n_users:
+        one: "You have muted %{count} User"
+        other: "You have muted %{count} users"
+      user_mute_explainer: "Private messages of muted users are moved into a separate inbox and you won't receive email notifications."
```

Comment:
We don't use the term "private messages" anywhere else in the UI, only "messages". So this could be confusing if someone is trying to work out if "private messages" are different from regular messages.

I think "private" can simply be removed in these two translations.